app-web/plugins/gatsby-source-github-all/utils/helpers.js

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2018 Province of British Columbia
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Created by Patrick Simonian
+*/
+/**
+ * returns an idempotent path based on a base path plus a digestable string that is hashed
+ * @param {String} source the string name of the original source
+ * @param  {Array} digestable 
+ * @returns {String} ie (/mypath, file.md) => /mypath/123dsfakjhdf
+ */
+const { TypeCheck } = require('@bcgov/common-web-utils'); // eslint-disable-line
+const path = require('path');
+const shorthash = require('shorthash');
+
+const createPathWithDigest = (base, digestable) => {
```

Comment:
A bit bunched up again. To make it more readable I'd consider a space before and after the 2nd `if` then one before the return.